frontend/common/src/components/SkillPicker/SkillPicker.tsx

```diff
@@ -168,17 +194,54 @@ const SkillPicker = ({
           },
         )}
       </p>
+      {skipId && (
+        <a
+          href={`#${skipId}`}
+          data-h2-visibility="base(invisible)"
+          data-h2-position="base:focus(static)"
+          data-h2-offset="base:focus(auto)"
+          data-h2-height="base:focus(auto)"
+          data-h2-width="base:focus(auto)"
+        >
+          {intl.formatMessage({
+            defaultMessage: "Skip list of skills",
+            id: "pg1S01",
+            description: "Link text to skip the list of add skill links",
+          })}
+        </a>
+      )}
+      {isInSkillList ? (
+        <p
+          aria-live="polite"
+          data-h2-visibility="base(invisible)"
+          id="skill-list-help"
+        >
+          {intl.formatMessage({
+            id: "tRYI0f",
+            defaultMessage:
+              "Press ctrl + shift + esc while navigating through the list of skills to skip directly to your chosen skills.",
```

Comment:
Mac users living in a Windows world will proabably know that ctrl -> command for them.  But does cmd+shift+esc work here too?

Reply:
It does not. However, we are working on the actual shortcut. @RM-1978 did some testing on a variety of screen readers last night and it turns out this interferes with a windows specific shortcut 😢